### DISCUSSION TITLE
Rotate an orientation about an rotation axis

### DISCUSSION THREAD

**Discussion by NovakEvana** [mtex-toolbox/mtex#1645] [Ask Anything] [answered]
Hi mtex community,

I tried to rotate a cube orientation about the misorientation of variant 14 of the K-S relationship which only has 3 positions as marked in red below. However,I always ended up with the pole figure with all variants like the 2nd image. I doubt that my symmetry was set wrongly somewhere. Thanks, everyone!

```
clear variables
close all;
cs = crystalSymmetry('m-3m');
ss = specimenSymmetry('triclinic');

setMTEXpref('xAxisDirection','south');
setMTEXpref('zAxisDirection','outofPlane');
h = Miller({0,0,1},cs);

%misorientation of variant 14
gamma_ori_1 = orientation.byMiller([-1 1 1],[0 -1 1],cs);
alpha_ori_1 = orientation.byMiller([0 1 1],[-1 1 -1],cs);
var_1 = inv(gamma_ori_1)*alpha_ori_1


cube = orientation.cube(cs,ss)
figure;
plotPDF(cube,h,'MarkerSize',10,'MarkerColor', 'b')
hold on;
cube = var_1 * cube
plotPDF(cube,h,'MarkerSize',10,'MarkerColor', 'r')
hold off;
```

![image](https://user-images.githubusercontent.com/117882416/201451605-5c0e9ac7-2f2a-4e83-9492-a4005e36d25f.png)

![image](https://user-images.githubusercontent.com/117882416/201451458-80626dfe-2fd7-499a-a5b1-3e8eb456b156.png)

**Comment by kilir** [comment, marked as answer]
Hi, 
an orientation applied from the left to a misorientation results in an orientation

```
cube = orientation.cube(cs,ss)
rcube = cube * var_1

figure;
plotPDF(cube,h,'MarkerSize',10,'MarkerColor', 'b')
hold on;
plotPDF(rcube,h,'MarkerSize',10,'MarkerColor', 'r')
hold off;
```
Cheers,
Rüdiger

**Reply by NovakEvana** [reply]
It works perfectly. I feel so grateful for having you support the community here! Thank you so much.
